Changelog.md

```diff
@@ -0,0 +1,8 @@
+
+# Changelog
+
+## GHC 9.4.7
+
+* cabal-install is now version 3.10
+* `prune-juice` has been removed. It is [deprecated](https://github.com/dfithian/prune-juice#readme).
```

Comment:
Since this is a breaking change, please explicitly note it, e.g.
```suggestion
* **Breaking:** `prune-juice` has been removed. It is [deprecated](https://github.com/dfithian/prune-juice#readme).
```